site/app/views/NavigationView.php

```diff
@@ -100,6 +100,8 @@ public function showGradeables($sections_to_list, $graded_gradeables, array $sub
         // DISPLAY ROOM SEATING (used to display room seating assignments)
         // ======================================================================================
         $display_room_seating = $this->core->getConfig()->displayRoomSeating();
+        $seating_only_for_instructor = $this->core->getConfig()->isSeatingOnlyForInstructor();
+        if(!$this->core->getUser()->accessAdmin() and $seating_only_for_instructor) $display_room_seating = false;
```

Comment:
Combine line 102 and 104 into just a single if statement branch so it's clearer what's happening:
```
if ($seating_only_for_instructor && !$this->core->getUser()->accessAdmin()) {
    $display_room_seating = false;
}
else {
    $display_room_seating = $this->core->getConfig()->displayRoomSeating();
}
```